### PR TITLE
Fixes the alignment bug on the popup for students completing a streak in exercises

### PR DESCRIPTION
**Pull request by jtamiace**

### CODE REVIEW

File: kalite/distributed/static/css/distributed/khan-lite.css [hunk 1]
```diff
@@ -924,6 +924,7 @@ to items you want to have the icon appear with.
 div#modal {
     position: absolute; /* makes the div go into a position that’s absolute to the browser viewing area */
     margin: auto;
+    padding-bottom: 50px;
```
Comment (MCGallaspy) [inline kalite/distributed/static/css/distributed/khan-lite.css]: I think this much padding is not appropriate in all cases:
ss

**Comment by MCGallaspy** [comment]
Is it possible to just grow the modal element's height with the content (instead of having a fixed height)?

**Comment by rtibbles** [comment]
I would like to suggest a different strategy here. KILL THE MODAL.

Seriously, we only added the modal for a very specific request in the Nalanda RCT, so, I would rather just watch it die, and use our normal messaging system instead.

**Comment by rtibbles** [comment]
(Also, the modal performance is terrible on low end tablets, with some students pressing multiple times to try to dismiss it, and all sorts of weird non-captured click event artifacts happening as a result).

**Comment by jamalex** [comment]
> KILL THE MODAL.

I agree. So much :crying_cat_face: here in watching students interact with the modal. Make it :game_die:!

**Comment by rtibbles** [comment]
Closing in favour of #3159

**Pull request closed without merging**